packages/p2p/src/components/order-details/order-details.scss

```diff
@@ -10,6 +10,7 @@ $card-width: 456px;
     @include tablet-up {
         min-height: $card-height;
         max-height: $card-height;
+        margin-bottom: 3rem;
```

Comment:
Added bottom margin for desktop screens:
## Before
![Screenshot 2022-05-19 at 4 39 41 PM](https://user-images.githubusercontent.com/103016120/169257197-95667d32-2c05-4b3f-b975-4e5eb26de68c.png)

## After
![Screenshot 2022-05-19 at 4 50 55 PM](https://user-images.githubusercontent.com/103016120/169257244-b91bbc33-4aff-4f0b-ab2b-017584644ea5.png)